hub/src/main/java/ch/prevo/open/hub/HubScheduler.java

```diff
@@ -37,7 +37,7 @@
     @Inject
     private HubService hubService;
 
-    @Scheduled(initialDelay = 10000, fixedDelay = 120000)
+    @Scheduled(initialDelayString = "${open.prevo.hub.config.scheduler.initialDelay.seconds}000", fixedDelayString = "${open.prevo.hub.config.scheduler.fixedDelay.seconds}000")
```

Comment:
The goal of the externalisation of the properties would have been that the person in charge of the demo could configure it as well. Therefore I'd add it to the readme and maybe provide the properties as well in the config folder.

Reply:
I was planning to make a demo specific config as part of #145. Perhaps this should be done as a command line parameter? Though I don't think the current users will change these settings